### PR TITLE
Fix no connection qmn

### PR DESCRIPTION
**Pull request by NikolaBorisov**
We found issue with qmn queries and {error, no_connection} 
Basically there will be just one error for N commands this PR tries to fix this.

### CODE REVIEW

**Comment by NikolaBorisov** [comment]
Fixes https://github.com/HalloAppInc/ecredis/issues/61

**Review by balusu-murali — APPROVED** [review]

**Review by mdonohue — APPROVED** [review]

File: src/ecredis.erl [hunk 1]
```diff
@@ -459,6 +465,18 @@ handle_ask(#query{command = Command, retries = Retries,
     end.
 
 -spec handle_error(Error :: atom(), Query :: #query{}) -> {[#query{}], [#query{}]}.
+handle_error(no_connection,
+        #query{query_type = qmn, cluster_name = ClusterName,
+            command = [Command | _], retries = Retries} = Query) ->
+    {ok, RemapNewVersion} = remap_cluster(Query),
+    case get_destination(ClusterName, Command) of
+        {ok, _Slot, Pid, Version} ->
+            {[], [Query#query{pid = Pid, version = Version, retries = Retries + 1}]};
```
Comment (vipin211) [inline src/ecredis.erl:474]: Will this execute all the qmn queries again?

Comment (NikolaBorisov) [reply src/ecredis.erl:474]: Basically a qmn query is split into N queries. Each query contains M redis commands that should execute on the same node. When we don't have a connection to this node we get just one {error, no_connection} and so here we remap and try to find a Pid again.

**Review by vipin211 — APPROVED** [review]

**Comment by NikolaBorisov** [comment]
I will try to add some tests for this one before I merge it

**Pull request merged**